src/docs/reference/modules/OpenId/README.md

```diff
@@ -49,7 +49,45 @@ Available settings are:
 - Enable Authorization Endpoint.
 - Enable Logout Endpoint.
 - Enable User Info Endpoint.
-- Allow Password Flow: It requires that the Token Endpoint is enabled. More info at <https://tools.ietf.org/html/rfc6749#section-1.3.3>
+- Allow Password Flow: It requires that the Token Endpoint is enabled. More info at <https://tools.ietf.org/html/rfc6749#section-1.3.3> 
+
+Typescript request example using axios:
+
+``` typescript 
+  const data: any =
+  {
+        username: username,
+        password: password,
+        // must read from config
+        client_id: 'client_id',
+        client_secret: 'client_secret',
+        grant_type: 'password',
+        scope: 'profile email phone roles'
+  }
+
+  let formData = []
+  for (let it in data) {
+    // eslint-disable-next-line no-prototype-builtins
+    if (data.hasOwnProperty(it)) {
+      formData.push(encodeURIComponent(it) + '=' + encodeURIComponent(data[it]))
+    }
+  } 
+  
+  var res = axios.request({
+        url: ApiUrl + '/connect/token',
+        method: 'post',
+        data: formData.join('&'),
+        headers: { 'Content-Type': 'application/x-www-form-urlencoded' }
+  })
+  
+  return res
+```
+Complete example at <https://github.com/JiZhouSoft/Jizhousoft.OrchardCore.Sample/blob/cd1fd0c42893c6d0607f8642451d1a98649b9769/vue-ts-admin/src/api/users.ts#L10>
+
+!!! note
+    For `client_secret` and `client_id` as confidential information, do not send it directly through the browser application. The sample code here is only used as a demonstration to send an authentication request through the client.
+Generally speaking, we should rewrite an `action` which only accept the username and password parameters, and then returns the verification result.
```

Comment:
Well, I don't think we should recommend that. In most cases, the best option is to use an interactive flow like code and declare the client as a public application.

Reply:
Sorry, I don't quite understand. Could you give me some more hints? About `declaring the client as a public application`

Reply:
@kevinchalet could you follow up here, please?

Reply:
I'm still super convinced having JS/TS code samples is the right approach. If links to the OAuth 2.0/OIDC specifications are not enough, maybe HTTP request samples would be better?

Reply:
I also thought about not using code, but I feel it will complicate the login process. Now I use oidc-client, which is very easy to use.

Reply:
So, what should we do here? Let's get to the finish line with this PR.

Reply:
I'd recommend closing it: we shouldn't encourage users to implement the protocol themselves by adding low-level JS/TS snippets. They are excellent client libraries out there 😃

Reply:
Maybe we should create a sample SAP project with front and back end separation
just like  `ts-oidc-client`

Reply:
Let's do that, then.